src/Logo/__tests__/Logo.test.js

```diff
@@ -5,17 +5,17 @@ import Logo from '..';
 
 describe('<Logo />', () => {
   it('should render withouth a problem', () => {
-    const render = shallowWithTheme(<Logo />);
-    expect(render).toMatchSnapshot();
+    const { container } = render(<Logo />);
+    expect(container.firstChild).toMatchSnapshot();
   });
 
   it('should render with custom width & height', () => {
-    const render = shallowWithTheme(<Logo width="300" height="80" />);
-    expect(render).toMatchSnapshot();
+    const { container } = render(<Logo width="300" height="80" />);
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
We want to avoid snapshot when is unnecessary. Instead, use `data-testid` from react-testing-library api.

```js
const logoNode = getByTestId('logo');

expect(logoNode).toHaveAttribute('width', '300')
expect(logoNode).toHaveAttribute('height', '300')
```